src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -181,6 +182,22 @@ trait BackendCall {
       Future.successful(ExecutionHash("", None))
   }
 
+  def pollingInitialInterval: FiniteDuration
+  def pollingMaxInterval: FiniteDuration
+  def pollingMultiplier: Double
+  def pollingInitialGap: FiniteDuration
```

Comment:
As far as I can tell, multiplier is always 1.1D. And the initial gap is probably 0 for everything except JES.